### PR TITLE
Update widget options ui

### PR DESCRIPTION
**Pull request by djnunez-aot**
*Issue #1103:*
https://github.com/bcgov/met-public/issues/1103

-Remove left padding

-Update icon colors

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).

### CODE REVIEW

File: met-web/src/components/engagement/form/EngagementWidgets/Documents/DocumentOptionCard.tsx [hunk 1]
```diff
@@ -71,7 +71,7 @@ const DocumentOptionCard = () => {
                         spacing={1}
                     >
                         <Grid item>
-                            <DescriptionOutlinedIcon sx={{ pl: '25px', fontSize: '5em' }} />
+                            <DescriptionOutlinedIcon htmlColor={'#707070'} sx={{ pl: '25px', fontSize: '5em' }} />
```
Comment (jadmsaadaot) [inline met-web/src/components/engagement/form/EngagementWidgets/Documents/DocumentOptionCard.tsx:74]: Can we take if from here
image

Comment (djnunez-aot) [reply met-web/src/components/engagement/form/EngagementWidgets/Documents/DocumentOptionCard.tsx]: done updated the info too & added it to palette thank you

File: met-web/src/components/engagement/form/EngagementWidgets/Documents/DocumentOptionCard.tsx [hunk 2]
```diff
@@ -71,7 +71,7 @@ const DocumentOptionCard = () => {
                         spacing={1}
                     >
                         <Grid item>
-                            <DescriptionOutlinedIcon sx={{ pl: '25px', fontSize: '5em' }} />
+                            <DescriptionOutlinedIcon color={'info'} sx={{ pl: '25px', fontSize: '5em' }} />
```
Comment (tiagograf-AOT) [inline met-web/src/components/engagement/form/EngagementWidgets/Documents/DocumentOptionCard.tsx:74]: no need for brackets color='info' should work

**Review by tiagograf-AOT — APPROVED** [review]

**Review by jadmsaadaot — APPROVED** [review]

File: met-web/src/components/engagement/form/EngagementWidgets/Documents/DocumentOptionCard.tsx (hunk 2 above)
Comment (djnunez-aot) [reply met-web/src/components/engagement/form/EngagementWidgets/Documents/DocumentOptionCard.tsx]: removed thank you for the suggestion

**Pull request merged**